packages/cypress/src/executors/cypress/cypress.impl.ts

```diff
@@ -67,7 +67,10 @@ export default async function cypressExecutor(
   options = normalizeOptions(options, context);
   // this is used by cypress component testing presets to build the executor contexts with the correct configuration options.
   process.env.NX_CYPRESS_TARGET_CONFIGURATION = context.configurationName;
-  let success;
+  // used to set babel settings for react CT.
+  process.env.NX_CYPRESS_COMPONENT_TEST =
```

Comment:
Can we try to make this a part of the react `nxComponentTestingPreset`?

Reply:
this does work! 🎉